dtmi/advantech/icam_500-1.json

```diff
@@ -0,0 +1,21 @@
+{
+  "@id": "dtmi:Advantech:ICAM_500;1",
+  "@type": "Interface",
+  "displayName": "ICAM-500",
+  "contents": [
+    {
+      "@type": "Component",
+      "name": "deviceinfo",
```

Comment:
deviceinfo here doesn't have consistent character casing compared to other usage (and our best practice guidelines) i.e. `deviceinfo` -> `deviceInfo`

Reply:
updated as suggestion